cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -99,9 +109,7 @@ <h3>
                           rows="6"
                           {%- if not value.was_it_helpful_text %}
                           required
-                          {% endif -%}>
-                    {{- form.comment.value() | default('', true) -}}
-                </textarea>
+                          {% endif -%}></textarea>
```

Comment:
Can we move `</textarea>` to its own line, like [this](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/v1/_includes/blocks/feedback.html#L104)?

Reply:
This adds prefix'd spaces to the textarea.

Reply:
Ah bummer, ok leave it.